ASUS/X556UR 1.0/oem.conf

```diff
@@ -0,0 +1,7 @@
+[OEM]
+Manufacturer=ASUS
+Product=Vivobook
+Version=X556UR 1.0
```

Comment:
Does the 1.0 mean anything here?

Reply:
I dont know for sure. I thought as the OS is reading model number as "X556UR 1.0" (as seen in neofetch), it will be more acccurate to detect the OEM info description if i added 1.0.

Reply:
Please correct me if it doesnt work like that :)

Reply:
@sanket-dm this repo/System Settings doesn't do anything automatically to match this data, it just has to be copied manually. That's an interesting idea, though—but there are so many devices out there, I don't know how practical that is. For now, though, just the regular model name makes sense without the `1.0`.